src/components/Common.js

```diff
@@ -44,6 +44,36 @@ export const SecondaryButton = styled(Button)`
   color: ${p => p.theme.colors.primary};
   border: 1px solid ${p => p.theme.colors.primary};
 `
+// todo: error message
+export const TextInput = styled.input.attrs({
+  type: 'text',
+})`
+  ${CardLike};  
```

Comment:
Which parts are we hoping to keep from CardLike?

Reply:
not sure yet, i am going to figure out how to implement the error message/state for text inputs